recipe/meta.yaml

```diff
@@ -9,14 +9,16 @@ source:
   sha256: e266aa0d9d9cddb876484a370b94f468248594a96ca0b6f87c21f969db2b8c5b
   patches:
     - force-protoc-executable.patch
+    - patches-grpc/0001-Link-libraries-privately.patch
+    - patches-grpc/0002-also-make-abseil-libs-private.patch  # [win]
```

Comment:
I didn't realize you linked only abseil as private

Reply:
I tried to see if including abseil unblocks the osx builds again. We definitely need this on windows. Again, I was just experimenting

Reply:
But the result was identical before that